packages/node_modules/@webex/plugin-logger/src/index.js

```diff
@@ -16,3 +17,7 @@ export {
   default,
   levels
 } from './logger';
+
+export {
```

Comment:
is there a reason not to have a simpler export, such `export const REDACTED = '[REDACTED]';`?

Reply:
Removed these exports altogether.